### PR TITLE
Adding front end design and front end developer definitions to the guide.

### PR DESCRIPTION
**Pull request by juliasolorzano**
This pull request is the initial start at helping to define the differences between front end design and development at 18F. These definitions relate to the work and research done around issue #123.

### CODE REVIEW

File: _includes/what-is-frontend.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
...
+
+**Front end designers** design, write and implement the presentational code base for websites and applications. They should have a clear understanding of design fundamentals and systems, such as interface style guides, responsive design, grid systems and front end frameworks. Front end designers should feel comfortable creating and implementing design systems using semantic HTML5, CSS/Sass and be able to assist in debugging this aspect of the code base.
+
+**Front end developers** architect, write and implement the funcational code base for websites and applications. They should have a clear understanding of client-side render and response, such as HTTP methods, API consumption and the browser loading/rendering pipeline. Front end developers should feel comfortable developing and implementing client-side interactions and frameworks using semantics HTML5, JavaScript and be able to assist in debugging, testing and performance optimization of the code base.
```
Comment (jbarnicle) [inline _includes/what-is-frontend.md]: _functional_ (funcational)

File: _includes/what-is-frontend.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+The front end guild did a series of exercises to determine the fundamental differences between the fron end design and front end developer roles at 18F. Using [some of our own research methods](https://methods.18f.gov), the front end guild came up with the following recommendations on knowing the difference between the two disciplines:
+
+**Front end designers** design, write and implement the presentational code base for websites and applications. They should have a clear understanding of design fundamentals and systems, such as interface style guides, responsive design, grid systems and front end frameworks. Front end designers should feel comfortable creating and implementing design systems using semantic HTML5, CSS/Sass and be able to assist in debugging this aspect of the code base.
```
Comment (maya) [inline _includes/what-is-frontend.md]: Needs comma after "write" and before "and."

**Comment by maya** [comment]
Looks good! The one thing I noticed absent is accessibility. Accessibility is everyone's job and _especially_ anyone touching the front end side of things. Can we add it in?

File: _includes/what-is-frontend.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+**Front end developers** architect, write and implement the functional code base for websites and applications. They should have a clear understanding of client-side render and response, such as HTTP methods, API consumption and the browser loading/rendering pipeline. Front end developers should feel comfortable developing and implementing client-side interactions and frameworks using semantics HTML5, JavaScript and be able to assist in debugging, testing and performance optimization of the code base.
```
Comment (maya) [inline _includes/what-is-frontend.md]: Needs comma after "write" and before "and."

**Comment by msecret** [comment]
Even though it was confirmed in research, it  bothers me that there's no CSS knowledge in the front end developers description. At 18F and elsewhere, I've found that CSS usually has way more tech debt then JS and requires developers to plan and architect in order to make successful

File: _includes/what-is-frontend.md (hunk 3 above)
Comment (xtine) [inline _includes/what-is-frontend.md]: /s/semantics/semantic

File: _includes/what-is-frontend.md (hunk 2 above)
Comment (xtine) [inline _includes/what-is-frontend.md]: Instead of just Sass, should we mention "CSS preprocessing tools" instead?

**Comment by juliasolorzano** [comment]
@maya @xtine thanks for your feedback! Updates made here cf4cb6f. Let me know if there are any other changes/suggestions!

**Comment by maya** [comment]
@juliaelman looking better! I think the last sentence of front end developers may have some grammar issues with the commas but I'd like someone on the content team to make sure.

**Comment by maya** [comment]
@juliaelman looks like this got merged. Did we still want to have someone from the content team review this?

**Comment by juliasolorzano** [comment]
@maya yup! 2ffbc5c

**Pull request merged**